CMakeLists.txt

```diff
@@ -95,16 +95,32 @@ endif()
 if (SKIP_PYBIND11)
   message(STATUS "SKIP_PYBIND11 set - disabling python bindings")
 else()
-  include(IgnPython)
-  find_package(PythonLibs QUIET)
-  if (NOT PYTHONLIBS_FOUND)
-    IGN_BUILD_WARNING("Python is missing: Python interfaces are disabled.")
-    message (STATUS "Searching for Python - not found.")
+  #include(IgnPython) TODO: allow to specify for what it should search and then
+  # the code below can be removed; e.g. pybind needs Interpreter and Development components
+  # see https://pybind11.readthedocs.io/en/stable/cmake/index.html#new-findpython-mode
+  if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
+    set(IGN_PYTHON_VERSION "3")
```

Comment:
I added `IGN_PYTHON_VERSION "3"` here to make it work. It turns out Bionic doesn't have pybind11 2.2 anyway, so this would only be useful for users that build pybind11 from source on Bionic.